tests/components/wallbox/test_number.py

```diff
@@ -34,23 +40,66 @@ async def test_wallbox_number_class(
         )
         mock_request.put(
             "https://api.wall-box.com/v2/charger/12345",
-            json=json.loads(json.dumps({CHARGER_MAX_CHARGING_CURRENT_KEY: 20})),
+            json=json.loads(
+                json.dumps(
+                    {
+                        CHARGER_MAX_CHARGING_CURRENT_KEY: 20,
+                    }
+                )
+            ),
             status_code=200,
         )
 
         await hass.services.async_call(
             "number",
             SERVICE_SET_VALUE,
             {
-                ATTR_ENTITY_ID: MOCK_NUMBER_ENTITY_ID,
+                ATTR_ENTITY_ID: MOCK_CHARGING_CURRENT_NUMBER_ENTITY_ID,
                 ATTR_VALUE: 20,
             },
             blocking=True,
         )
     await hass.config_entries.async_unload(entry.entry_id)
 
 
-async def test_wallbox_number_class_connection_error(
+async def test_wallbox_energy_price_number_class(
+    hass: HomeAssistant, entry: MockConfigEntry
+) -> None:
+    """Test wallbox sensor class."""
+
+    await setup_integration(hass, entry)
+
+    with requests_mock.Mocker() as mock_request:
+        mock_request.get(
+            "https://user-api.wall-box.com/users/signin",
+            json=authorisation_response,
+            status_code=200,
+        )
+        mock_request.put(
+            "https://api.wall-box.com/v2/charger/12345",
```

Comment:
you are mocking the wrong http address, see above.

Reply:
Should be fixed now.